logistics_project/apps/malawi/nag.py

```diff
@@ -129,9 +130,9 @@ def _get_hsas_ready_for_nag(warning_time, extra_filter_params={}):
                                         nag_type=Reports.REC,
                                         **extra_filter_params).count():
 
-                if not ProductReport.objects.filter(report_type__code=Reports.REC,
-                                                 report_date__range=[req.responded_on,
-                                                                     now]).exists():
+                if not ProductReport.objects.filter(supply_point=req.supply_point,
```

Comment:
how bout that. nice catch. i wonder how significantly this impacts receipt reporting rates.